content/projects/java-specific/streams/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+content_type: project
+flavours:
+- java
+prerequisites:
+  hard:
+  - projects/semitone-challenge/basic-algorithm
+  - topics/java-specific/streams
+ready: true
+submission_type: repo
+tags:
+- data-structures
+- array-manipulation
+title: Java Streams
+---
+
+Remember your semitone challenge project ***(see prerequisites above)*** well for this project we are about to make that project more cooller.
+
+The requirements here is that you should have already submitted your semitone project with basic algorith and now we are going to use other functions to process those data structures.
+
+#### Instructions:
+
+Pretty simple just use atleast three operators of streams to refactor your implementation of the semitone challenge, make sure you test your logic aspecially the paths that use streams.
+
+
+#### Instructions to the review
+- Ensure they used three or more stream function
+
+- Ensure they have tests that cover logic where streams are used
```

Comment:
To keep consistency with the rest of the syllabus, I think this part should be at the very end.